kentikapi/internal/api_payloads/saved_filter.go

```diff
@@ -79,16 +80,46 @@ func SavedFilterToPayload(sf models.SavedFilter) SavedFilterPayload {
 	}
 }
 
-type FiltersPayload struct {
+func SavedFilterToCreatePayload(sf models.SavedFilter) CreateSavedFilterRequest {
+	return CreateSavedFilterRequest{
+		savedFilterPayload: savedFilterPayload{
```

Comment:
Please use SavedFilterToPayload() here to remove duplication.
Alternatively remove  SavedFilterToCreatePayload() and SavedFilterToUpdatePayload() function and use only SavedFilterToPayload() everywhere.

For SavedFilterToUpdatePayload function too.